README.md

```diff
@@ -142,20 +168,5 @@ nohup bash run_local_singularity.sh &
 
 > ✨ Note: You can use any one of the: `nohup`, `screen` or `tmux`.
 
-## Testing
-To ensure that the version is working properly you can test it by:
```

Comment:
To ensure that the *code* is working... version would raise question as to what the version is and how it relates to others